meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files/imx8qm-mek/full_image.uuu.in

```diff
@@ -21,7 +21,7 @@ SDPV: jump
 
 FB: ucmd setenv fastboot_dev mmc
 FB: ucmd setenv mmcdev 0
-FB: ucmd mmc dev ${mmcdev} 1; mmc erase 0 0x3FFE
```

Comment:
We don't need to do that for iMX8QM-MEK, as Toradex config block is not used here

Reply:
same for `imx8qm-mek/bootloader.uuu.in`

Reply:
Indeed, nice catch.

Reply:
@ricardosalveti that was @MrCry0 who initially noticed that :)

Reply:
> @ricardosalveti that was @MrCry0 who initially noticed that (we even discussed that in Slack before) :)

Thanks to @igoropaniuk, he noted there were still no my comments :)

It was funny, I wrote comments, but as a part of a review and forgot to submit it :-D

Reply:
Dropped this patch.  Good catch.